bin/os_tf.sh

```diff
@@ -32,10 +32,12 @@ md5sum "$kubenow_image_name".qcow2 >"$kubenow_image_name".qcow2.md5
 
 # Uploading the new image format to the AWS S3 bucket. Previous copy will be overwritten.
 echo "Uploading new image format into AWS S3 bucket: kubenow-us-east-1 ..."
-aws s3 cp "$kubenow_image_name".qcow2 s3://kubenow-us-east-1 --region us-east-1 --acl public-read --quiet
-aws s3 cp "$kubenow_image_name".qcow2.md5 s3://kubenow-us-east-1 --region us-east-1 --acl public-read --quiet
+bucket1_region=$(echo "$AWS_BUCKET1_URL" | awk -F "/" '{{ print $2 $3 $4 }}' | sed -e 's/kubenow-//')
+aws s3 cp "$kubenow_image_name".qcow2 "$AWS_BUCKET1_URL" --region "$bucket1_region" --acl public-read --quiet
+aws s3 cp "$kubenow_image_name".qcow2.md5 "$AWS_BUCKET1_URL" --region "$bucket1_region" --acl public-read --quiet
```

Comment:
Not using hard-coded URL anymore. This should make easier future migration and/or adding new buckets.